kfserving/kfserving-install/base/config-map.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+data:
+  credentials: |-
+    {
+       "gcs" : {
+           "gcsCredentialFileName": "gcloud-application-credentials.json"
+       },
+       "s3" : {
+           "s3AccessKeyIDName": "awsAccessKeyID",
+           "s3SecretAccessKeyName": "awsSecretAccessKey"
+       }
+    }
+  frameworks: |-
+    {
+        "tensorflow": {
+            "image": "tensorflow/serving"
+        },
+        "sklearn": {
+            "image": "gcr.io/kfserving/sklearnserver"
+        },
+        "pytorch": {
+            "image": "gcr.io/kfserving/pytorchserver"
+        },
+        "xgboost": {
+            "image": "gcr.io/kfserving/xgbserver"
+        },
+        "tensorrt": {
+            "image": "nvcr.io/nvidia/tensorrtserver"
+        }
+    }
+kind: ConfigMap
+metadata:
+  name: kfservice-config
+  namespace: kfserving-system
```

Comment:
if you do something like the following
```
  - kustomizeConfig:
      parameters:
      - name: namespace
        value: kfserving-system
      repoRef:
        name: manifests
        path: kfserving/kfserving-install
    name: kfserving-install
```

in a config file that deploys kfserving (see [kfctl_gcp_iap.yaml](https://github.com/kubeflow/kubeflow/blob/master/bootstrap/config/kfctl_gcp_iap.yaml) for example)
then you can remove all namespace lines including I believe

```
subjects:
- kind: ServiceAccount
  name: default
  namespace: kfserving-system
```